ntc_templates/templates/extreme_exos_show_ports_no-refresh.textfsm

```diff
@@ -0,0 +1,20 @@
+Value PORT (\d+)
+Value NAME (\S*)
+Value VLAN (\S*)
+Value STATE ([DEFL])
+Value LINKSTATE ([ARLDdB|NP])
+Value SPEED (\d*)
+Value DUPLEX ([HALF|FULL]*)
+
+Start
+  ^Port.*
+  ^.*String.*VLANs.*State\sState\sActual\sActual
+  ^\=+
+  ^${PORT}\s+${NAME}\s+${VLAN}\s+${STATE}\s+${LINKSTATE}\s+${SPEED}\s+${DUPLEX} -> Record
```

Comment:
I think you need to use a combo of look forwards and behind, otherwise there is no way to tell the difference between `name` and `vlan` as they both have `\S*`. Which results in 

```
  - port: "9"
    name: "Default"
    vlan: ""
    state: "E"
    linkstate: "R"
    speed: ""
    duplex: ""
```

A common strategy is to list out each combo, something like: (the spacing is not verified, just giving hints).
```
Value NAME (\S+)
Value VLAN (\S+)

  ^${PORT}\s+${NAME}\s+${VLAN}\s+${STATE}\s+${LINKSTATE}\s+${SPEED}\s+${DUPLEX} -> Record
  ^${PORT}\s{22}${VLAN}\s+${STATE}\s+${LINKSTATE}\s+${SPEED}\s+${DUPLEX} -> Record
  ^${PORT}\s{21}${VLAN}\s+${STATE}\s+${LINKSTATE}\s+${SPEED}\s+${DUPLEX} -> Record
  ^${PORT}\s{7}${NAME}\s+${STATE}\s+${LINKSTATE}\s+${SPEED}\s+${DUPLEX} -> Record
  ^${PORT}\s{6}${NAME}\s+${STATE}\s+${LINKSTATE}\s+${SPEED}\s+${DUPLEX} -> Record
```

Reply:
Ah, my bad! Will fix.

Reply:
Honestly, well done for a first PR!

Reply:
Thanks,
Made the suggested update. Added stack support as well.